doc/source/whatsnew/v1.1.0.rst

```diff
@@ -13,6 +13,15 @@ including other versions of pandas.
 Enhancements
 ~~~~~~~~~~~~
 
+.. _whatsnew_110.specify_missing_labels:
+
+KeyErrors raised by loc specify missing labels
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Previously, if labels were missing for a loc call, a Key Error was raised stating that this was no longer supported.
+
+Now the error message also includes a list of the missing labels (max 10 items, display width 80 characters).
+
```

Comment:
can u add the issue number here

Reply:
done

Reply:
can you do it like other issue references, e.g. ```:issue:`34272` ```

Reply:
Sure, done.

Reply:
@jreback - sure, this now done.